modules/Bio/EnsEMBL/Analysis/Hive/Config/RepeatMasking.pm

```diff
@@ -0,0 +1,549 @@
+########################
+
+    # The following might not be known in advance, since the come from other pipelines
+    # These values can be replaced in the analysis_base table if they're not known yet
+    # If they are not needed (i.e. no projection or rnaseq) then leave them as is
+
+    'pipe_db_name'                  => $self->o('dbowner').'_'.$self->o('production_name').'_pipe_'.$self->o('release_number'),
+    'dna_db_name'                   => $self->o('dbowner').'_'.$self->o('production_name').'_core_'.$self->o('release_number'),
+
+    'reference_db_name'            => $self->o('dna_db_name'),
+    'reference_db_server'          => $self->o('dna_db_server'),
+    'reference_db_port'            => $self->o('dna_db_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'              => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+    'production_db_server'         => 'mysql-ens-meta-prod-1',
+    'production_db_port'           => '4483',
+    
+    databases_to_delete => ['reference_db'],
+
+    
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps                  => catdir($self->o('output_path'), 'genome_dumps'),
+    # This one is used by most analyses that run against a genome flatfile like exonerate, genblast etc. Has slice name style headers. Is softmasked
+    softmasked_genome_file        => catfile($self->o('genome_dumps'), $self->o('species_name').'_softmasked_toplevel.fa'),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+    faidx_genome_file             => catfile($self->o('genome_dumps'), $self->o('species_name').'_toplevel.fa'),
+    # This one is a cross between the two above, it has the seq_region name header but is softmasked. It is used by things that would both want to skip using the dna table and also want to avoid the repeat_feature table, e.g. bam2introns
+    faidx_softmasked_genome_file  => catfile($self->o('genome_dumps'), $self->o('species_name').'_softmasked_toplevel.fa.reheader'),
+
+    full_repbase_logic_name => "repeatmask_repbase_".$self->o('repbase_logic_name'),
+
+    'min_toplevel_slice_length'   => 250,
+
+    'repeatmodeler_logic_name'    => 'repeatmask_repeatmodeler',
+    
+    ensembl_analysis_script           => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    sequence_dump_script              => catfile($self->o('ensembl_analysis_script'), 'sequence_dump.pl'),
+    
+    ensembl_misc_script        => catdir($self->o('enscode_root_dir'), 'ensembl', 'misc-scripts'),
+    repeat_types_script        => catfile($self->o('ensembl_misc_script'), 'repeats', 'repeat-types.pl'),
```

Comment:
This can be removed, it is not used in the config. I agree I should have probably have added it to the pipeline but I tried to follow the flow of the pipeline.

Reply:
Changes effected